capellambse_context_diagrams/serializers.py

```diff
@@ -226,7 +226,7 @@ def get_styleclass(obj: common.GenericElement) -> str:
     """Return the styleclass for a given `obj`."""
     styleclass = obj.__class__.__name__
     styleclass = (
-        aird._semantic.STYLECLASS_LOOKUP.get(styleclass, (styleclass, None))[0]
+        aird.STYLECLASS_LOOKUP.get(styleclass, (styleclass, None))[0]
```

Comment:
This is odd. The `STYLECLASS_LOOKUP` is not supposed to be useful outside of the AIRD parser, which is why it's not exported from there. What exactly are you after by calling into it? Wouldn't it make more sense to move the relevant part to the more generic `capellambse.diagram` module?

Reply:
Same problem still exists, and arguably just got worse. Now you just reverted to using a random private dictionary from some 3rd-party module. Nobody's going to bat an eye when changing it there, which would break this entire extension (this function looks important).

And my questions from earlier remain unanswered, which makes me sad.

Reply:
Nothing got worse. I just reverted to the state from before. Also `STYLECLASS_LOOKUP` isn't random, it's a fully deterministic variable, but private tho.

We'll migrate `get_styleclass` to a public function in capellambse.